fighthealthinsurance/generate_appeal.py

```
@@ -925,6 +925,137 @@ def _shed_context(
+    """
+    if not model_name:
+        return None
+    backends = ml_router.models_by_name.get(model_name)
+    if not backends:
+        return None
+    limits: List[int] = []
+    for backend in backends:
+        try:
+            limits.append(int(backend.get_max_context()))
+        except Exception:
+            continue
```

Ruff (0.15.20)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)